### PR TITLE
chore: rename master branch references to main

### LINKED ISSUE #582: Shall we rename the master branch to main?
Hey @piotr-oles!

How would you feed about us renaming the `master` branch to `main`?

## Feature motivation

Many communities, both on GitHub and in the wider Git community, are considering renaming the default branch name of their repository from `master`. GitHub is gradually renaming the default branch of their own repositories from `master` to `main`.

## Feature description

Renaming the branch.

## Feature implementation

Following the instructions here is super simple: https://github.com/github/renaming

I did this on [`ts-loader`](https://github.com/TypeStrong/ts-loader) today and it was super straightforward.  Renaming the branch and then copy-pasting the command to change my local branch and I was done.  Would recommend!

### PR DESCRIPTION
**Pull request by johnnyreilly**
Hey @piotr-oles!

If you're happy with this, I'd suggest we follow the instructions here: https://github.com/github/renaming

And *then* merge this PR.  I'm happy to do this, but I'll hold off until you've approved this PR first.  Or if you'd like to handle it - that's fine too!

✅ Closes: #582

### CODE REVIEW

File: src/formatter/types/babel__code-frame.ts [hunk 1]
```diff
@@ -1,5 +1,5 @@
 // Base on the type definitions for @babel/code-frame 7.0
-// Project: https://github.com/babel/babel/tree/master/packages/babel-code-frame, https://babeljs.io
+// Project: https://github.com/babel/babel/tree/main/packages/babel-code-frame, https://babeljs.io
```
Comment (johnnyreilly) [inline src/formatter/types/babel__code-frame.ts:2]: although this is referencing another repo - it turns out that repo has already renamed from `master` to `main` - so this is not a mistake!

**Review by piotr-oles — APPROVED** [review]
Looks good, thanks! Feel free to update the config :)

**Comment by johnnyreilly** [comment]
Okay just did the rename.

**Comment by johnnyreilly** [comment]
If you have a local clone, you can update it by running:

```
git branch -m master main
git fetch origin
git branch -u origin/main main
git remote set-head origin -a
```

**Comment by johnnyreilly** [comment]
Merged! We should be done...

**Pull request merged**